docs/hugo/content/introduction/annotations.md

```diff
@@ -0,0 +1,27 @@
+# Annotations understood by the operator
+
+## Annotations specified by the user
+
+Note that unless otherwise specified, allowed values are _case sensitive_ and should be provided in lower case.
+
+1. `serviceoperator.azure.com/reconcile-policy`: Specifies the reconcile policy to use. Allowed values are:
+    - `run`: The operator performs all actions as normal. This is the default if no annotation is specified.
+    - `skip`: All modification actions on the backing Azure resource are skipped. GETs are still issued to ensure that the resource
+    exists. If the resource doesn't exist, the `Ready` condition will show a `Warning` state with the details of the missing resource
+    until the resource is created. If the resource is deleted in Kubernetes, it is _not_ deleted in Azure. In REST API terminology, 
+    PUT and DELETE are skipped while GET is allowed.
+    - `skip-delete`: Modifications are pushed to the backing Azure resource, but if the resource is deleted in Kubernetes, 
```

Comment:
I think `abandon` or `detach` might be good terms?

Reply:
Reusing the term `skip` seems to leave things open for misinterpretation (not that we can ever completely avoid that).

In the context of reconciler behaviour, how about `never-delete` ?

Reply:
Ok I like: `reconcile-policy:` `manage|skip|detach-on-delete` I think. Thoughts?